### ISSUE TITLE
[Documentation] Adjust english translation of templates.rst

### ISSUE THREAD

**Issue by AxxL** [mapbender/mapbender-documentation#70]
File: Wie werden eigene Style-Vorlagen (templates) erzeugt?

Commit: https://github.com/mapbender/mapbender-documentation/commit/8fc01122d96c00c48214b79a5391c7448b1c8031

The german file has some changes that should be translated to english.

English file to translate: en/customization/templates.rst

German file that has to be translated: de/customization/templates.rst

**Comment by fhaefker** [comment]
Hi @CharlyUnicorn

Please give me an overview of whether action to remedy this problem is required or whether it can be closed

**Comment by rschwammborn** [comment]
Not @CharlyUnicorn , but let me present my suggestions regarding this issue:

1. The different languages of templates.rst are mostly in sync, so @AxxL s main reason to open this issue a long time ago is already gone.

1. Nethertheless, the two files are still a little messy. This is probably why @eschuettler made the following PR: https://github.com/mapbender/mapbender-documentation/pull/182 

1. Only regarding the issue #133, I also changed the template files and moved a not needed section (introduction to sidepane usage) to another page. For that, we have another PR: https://github.com/mapbender/mapbender-documentation/pull/192. Check the mentioned issue above for further details.

1. Lastly: I guess @jkrey just used the wrong channel to communicate about his further to-do on a similar job.

### Recommendation:

Because of the files already being in-sync, I think it's okay to close this issue. However, the remaining work regarding the template files should be merged into master. IMO, the right order is as follows: Because #182 has more in-depth changes, it should be merged first. #192 can follow right after (or needs readjustment because of merge conflicts). Also, the PRs I'm talking about here are still in review, so you still have to wait for approval.

Hope that helps.

**Comment by fhaefker** [comment]
Hi @rschwammborn

sounds legit

I will close this issue as recommended by you.

Please continue as proposed by you -> first #182 than #192

Best Regards
Frederik
